test/util.ts

```diff
@@ -1092,33 +1093,6 @@ describe('common/util', () => {
       };
     }
 
-    describe('callback mode', () => {
-      it('should pass the default options to retryRequest', (done) => {
-        retryRequestOverride = testDefaultRetryRequestConfig(done);
-        // tslint:disable-next-line:no-any
-        util.makeRequest(reqOpts, {request: fakeRequest as any}, () => {});
-      });
-
-      it('should allow turning off retries to retryRequest', (done) => {
-        retryRequestOverride = testNoRetryRequestConfig(done);
-        util.makeRequest(reqOpts, noRetryRequestConfig, () => {});
-      });
-
-      it('should override number of retries to retryRequest', (done) => {
-        retryRequestOverride = testCustomRetryRequestConfig(done);
-        util.makeRequest(reqOpts, customRetryRequestConfig, () => {});
-      });
-
-      it('should return the instance of retryRequest', () => {
```

Comment:
This was the one test that wasn't duplicated, so I just moved it down with the rest.